modules/API/pages/gsql-endpoints.adoc

```diff
@@ -3134,6 +3134,11 @@ Sample Response::
 
 This endpoint is used to intepret query. Please note that calling this endpont need to set 'Content-Type: text/plain' in header.
 
+For interpret query, we can declare workload queue in 2 ways:
```

Comment:
Please improve the grammar. This whole page was originally published without the grammar being proofread.